tools/install_runner_prerequisites.sh

```diff
@@ -104,6 +104,7 @@ function install_packages {
       'library/postgresql-13'
       'pkg-config'
       'brand/omicron1/tools'
+      'brand/sparse'
```

Comment:
The zone created by the outgoing `create_virtual_hardware.sh` uses an `omicron1` zone (set in the `softnpu-zone.txt`). Do we _need_ to use a `sparse` zone with the new softnpu tooling?

Reply:
My memory fails me, but I remember troubleshooting some issue with @jclulow that caused me to switch it from `sparse` to `omicron1`, I think at the time it was because the `sparse` zone relied on `pkg` which [wasn't available in the ramdisk image in CI](https://buildomat.eng.oxide.computer/wg/0/details/01GNTHDT85XRMV9AT5XXYMB3RV/GVMtns2wc4vVF7yOT2BPbH4CrZYoEWYw6K0ktr4PNeskUUQJ/01GNTHE62NHH8DPW2R2MTHVE2V#S332). It seems this may have changed though, because your `deploy` task was successful? 🤔

Reply:
FWIW, the issues that lead me to suggest **omicron1** instead of **sparse** haven't changed, so if it is somehow working in the deploy task ramdisk environment, and it depends on the sparse brand, I don't know how! :D

Reply:
Yeah this is an artifact from previous attempts, will remove.